spdx-utils/src/main/kotlin/model/SpdxDocument.kt

```diff
@@ -138,5 +138,13 @@ data class SpdxDocument(
         require(dataLicense.isNotBlank()) { "The data license must not be blank." }
 
         require(documentNamespace.isNotBlank()) { "The document namespace must not be blank." }
+
+        require(
+            (relationships.any { it.relationshipType == SpdxRelationship.Type.DESCRIBES })
+                    || documentDescribes.isNotEmpty()
+        ) {
+            "The document must either have at least one relationship of type 'DESCRIBES' " +
+                    "or contain the 'documentDescribes' field."
+        }
```

Comment:
Let's rewrite this slightly more readable as

        val hasDescribesRelationship = relationships.any { it.relationshipType == SpdxRelationship.Type.DESCRIBES }
        require(hasDescribesRelationship || documentDescribes.isNotEmpty()) {
            "The document must either have at least one relationship of type 'DESCRIBES' or contain the " +
                    "'documentDescribes' field."
        }